demo/demo/tests/conftest.py

```diff
@@ -139,6 +139,14 @@ def wrapped(**kwargs):
     return wrapped
 
 
+@pytest.fixture
+def refreshes(user, resource) -> Callable:
+    @wraps(get_resource_refresh_moments)
+    def wrapped(**kwargs) -> Optional[datetime]:
+        return get_resource_refresh_moments(user=user, **kwargs).get(resource, None)
+    return wrapped
```

Comment:
Nice

Reply:
It's stolen from your tests. I've read it and was like "Nice!" and just used it again :p